pkg/networkservice/chains/forwarder/server.go

```diff
@@ -73,23 +72,29 @@ type xconnectNSServer struct {
 }
 
 // NewServer - returns an implementation of the xconnectns network service
-func NewServer(ctx context.Context, name string, authzServer networkservice.NetworkServiceServer, tokenGenerator token.GeneratorFunc, clientURL *url.URL, vppConn Connection, tunnelIP net.IP, tunnelPort uint16, dialTimeout time.Duration, domain2Device map[string]string, clientDialOptions ...grpc.DialOption) endpoint.Endpoint {
-	nseClient := registryclient.NewNetworkServiceEndpointRegistryClient(ctx, clientURL,
+func NewServer(ctx context.Context, tokenGenerator token.GeneratorFunc, vppConn Connection, tunnelIP net.IP, options ...Option) endpoint.Endpoint {
+	opts := &forwarderOptions{
```

Comment:
Missed default parameters for:

	authorizeServer -> null server
	clientURL -> default connectTo
	dialTimeout   ->  200ms
	domain2Device  ->empty map
	statsOpts       []stats.Option
	cleanupOpts     []cleanup.Option
	vxlanOpts       []vxlan.Option
	dialOpts        []grpc.DialOption

Reply:
I thought it was the responsibility of each individual chain element. Forwarder just passes options. Otherwise, it will turn out that at each iteration we will set new default parameters:
```
func AnyOtherNewServer(opts ...Options) {
  /* set default parameters */
  ...
  forwarder.NewServer(..., forwarder.WithOption(opts.AnyOption)) <- new default parameters inside forwarder.NewServer
  ...
}
```
Name is set, because we want to keep consistency inside the forwarder (used in the `client` and `endpoint` chain).
And, do we have a constant for `dialTimeout` or `clientURL`?

@denis-tingaikin Any thoughts?

Reply:
@glazychev-art option means that the value is not required. I don't think that clientURL== nil and dialTimeout == 0 is good values by default.